src/lib/db/feature-strategy-store.ts

```diff
@@ -361,10 +418,25 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
                         environments: [
                             FeatureStrategiesStore.getEnvironment(r),
                         ],
+                        tags: [],
                     };
+                    if (this.isUnseenTag(acc[r.feature_name], r)) {
```

Comment:
why do it need to be unseenTag?

Reply:
See: https://github.com/Unleash/unleash/pull/2396#issuecomment-1313539524